app/src/main/java/in/testpress/testpress/core/Constants.java

```diff
@@ -82,6 +82,8 @@ private Http() {}
 
         public static final String CHAPTERS_PATH =  "/api/v2.2.1/chapters/";
 
+        public static final String PRESIGNED_SSO_URL =  "/api/v2.3/presigned_sso_url/";
```

Comment:
Refactor this name to follow the above naming conventions used in this file and this is not presigned sso url

Reply:
Changed